### ISSUE TITLE
Major client bug in latest releases. 

### ISSUE THREAD

**Issue by Zelioth** [mrh0/createaddition#365]
**Describe the bug**
Client does not start past loading

**Additional context**
- Mod version: 20221140a
- Forge version: 43.1.47
- Create version: 0.5.0f

[crash-2022-11-04_20.16.52-fml.txt](https://github.com/mrh0/createaddition/files/9942684/crash-2022-11-04_20.16.52-fml.txt)
[crash-2022-11-04_20.19.06-fml.txt](https://github.com/mrh0/createaddition/files/9942685/crash-2022-11-04_20.19.06-fml.txt)

**Comment by littlepianist** [comment]
I get the same bug

**Comment by Peppe-KSP** [comment]
Same error:
`[05Nov2022 07:15:03.975] [Worker-Main3/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/]: Exception caught during firing event: null
	Index: 1
	Listeners:
		0: NORMAL
		1: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@2eb7fc5
		2: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@133243d6
		3: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@11bac407
		4: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@5701b1e8
		5: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@ecde99a
		6: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@f603766
		7: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@48bd4071
		8: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@12e9267f
		9: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@73099c81
		10: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@7d5c4e81
		11: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@5fc02793
		12: net.minecraftforge.eventbus.EventBus$$Lambda$4731/0x0000000801867c20@44138ca
java.lang.ExceptionInInitializerError
	at TRANSFORMER/createaddition@1.19.2-20221104a/com.mrh0.createaddition.CreateAddition.doClientStuff(CreateAddition.java:125)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:315)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:296)
	at LAYER PLUGIN/javafmllanguage@1.19.2-43.1.47/net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:107)
	at LAYER PLUGIN/fmlcore@1.19.2-43.1.47/net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:122)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
Caused by: java.lang.RuntimeException: PartialModel 'createaddition:entity/liquid_hat' loaded after ModelEvent.RegisterAdditional
	at TRANSFORMER/flywheel@0.6.7-8/com.jozufozu.flywheel.core.PartialModel.<init>(PartialModel.java:31)
	at TRANSFORMER/createaddition@1.19.2-20221104a/com.mrh0.createaddition.index.CAPartials.entity(CAPartials.java:17)
	at TRANSFORMER/createaddition@1.19.2-20221104a/com.mrh0.createaddition.index.CAPartials.<clinit>(CAPartials.java:10)
	... 14 more

[05Nov2022 07:15:03.975] [Worker-Main3/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/LOADING]: Caught exception during event FMLClientSetupEvent dispatch for modid createaddition
java.lang.ExceptionInInitializerError: null
	at com.mrh0.createaddition.CreateAddition.doClientStuff(CreateAddition.java:125) ~[createaddition-1.19.2-20221104a.jar%23176!/:1.19.2-20221104a]
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260) ~[eventbus-6.0.3.jar%2379!/:?]
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252) ~[eventbus-6.0.3.jar%2379!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.3.jar%2379!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.3.jar%2379!/:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:107) ~[javafmllanguage-1.19.2-43.1.47.jar%23279!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:122) ~[fmlcore-1.19.2-43.1.47.jar%23278!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source) [?:?]
Caused by: java.lang.RuntimeException: PartialModel 'createaddition:entity/liquid_hat' loaded after ModelEvent.RegisterAdditional
	at com.jozufozu.flywheel.core.PartialModel.<init>(PartialModel.java:31) ~[flywheel-forge-1.19.2-0.6.7-8.jar%23286!/:0.6.7-8]
	at com.mrh0.createaddition.index.CAPartials.entity(CAPartials.java:17) ~[createaddition-1.19.2-20221104a.jar%23176!/:1.19.2-20221104a]
	at com.mrh0.createaddition.index.CAPartials.<clinit>(CAPartials.java:10) ~[createaddition-1.19.2-20221104a.jar%23176!/:1.19.2-20221104a]
	... 14 more`

**Comment by YOYOK9** [comment]
Rolling back to version createaddition-1.19.2-20221024a stopped it from crashing, both 20221104a & 20221103a produced an error. 

Possibly a confliction or issue with flywheel? 

Latest: https://gist.github.com/YOYOK9/4f63414d0084086021eed0ee878102cd
crash report: https://gist.github.com/YOYOK9/99d6b44ca9a3c01991bf5f81f39bd349

----------------------------------------------------
I did test the mod on its own as well, worked with no issues. Rolled back all the mods I updated in my pack and tried just updating create additions, game crashes.
